### PR TITLE
Add www.presencedb.com support

### PR DESCRIPTION
**Pull request by vascYT**

### CODE REVIEW

File: websites/P/Plex/dist/metadata.json [hunk 1]
```diff
@@ -12,9 +12,9 @@
   "url": "www.plex.tv",
   "version": "1.7.3",
   "logo": "https://i.imgur.com/FPxlGGE.png",
-  "thumbnail": "https://i.imgur.com/DQvN7Nx.png",
-  "color": "#d78c14",
-  "tags": ["video", "music"],
+  "thumbnail": "https://files.vasc.dev/source/DU7ayUCih.png",
```
Comment (EncryptedDev) [inline websites/P/Plex/dist/metadata.json:15]: Must be on Imgur

File: websites/P/PresenceDB/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        "id": "346977366569910274"
+    },
+    "service": "PresenceDB",
+    "description": {
+        "en": "Tracks and saves the playtime of your Discord activities.",
+        "de": "Verfolgt und speichert die Spielzeit Ihrer Discord Aktivitäten."
+    },
+    "url": ["www.presencedb.com", "preview.presencedb.com"],
+    "version": "1.0.0",
+    "logo": "https://www.presencedb.com/logo.png",
```
Comment (EncryptedDev) [inline websites/P/PresenceDB/dist/metadata.json:14]: Must be on imgur

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Look at my review

**Comment by ririxidev** [comment]
Site not 2 months old.

**Comment by vascYT** [comment]
See you in 2 Months then 👋

**Pull request closed without merging**